src/Symfony/Component/Validator/Tests/Constraints/AtLeastOneOfValidatorTest.php

```diff
@@ -258,6 +262,36 @@ public function testNestedConstraintsAreNotExecutedWhenGroupDoesNotMatch()
 
         $this->assertCount(1, $violations);
     }
+
+    public function testTranslatorIsCalledOnConstraintBaseMessage()
+    {
+        $translator = new class() implements TranslatorInterface, LocaleAwareInterface {
+            use TranslatorTrait;
+
+            public function trans(?string $id, array $parameters = [], string $domain = null, string $locale = null): string
+            {
+                if ($id === 'This value should satisfy at least one of the following constraints:') {
```

Comment:
let's add another case to also translate "This value should be null."?

Reply:
This should be it! Also renamed the test case to be more relevant 🙂